domain/src/test/java/org/sagebionetworks/research/domain/navigation/TestStep.java

```diff
@@ -0,0 +1,8 @@
+package org.sagebionetworks.research.domain.navigation;
+
+import org.sagebionetworks.research.domain.step.Step;
+import org.sagebionetworks.research.domain.task.navigation.strategy.StepNavigationStrategy.BackStepStrategy;
+import org.sagebionetworks.research.domain.task.navigation.strategy.StepNavigationStrategy.NextStepStrategy;
+import org.sagebionetworks.research.domain.task.navigation.strategy.StepNavigationStrategy.SkipStepStrategy;
+
+public interface TestStep extends Step, NextStepStrategy, BackStepStrategy, SkipStepStrategy { }
```

Comment:
I just discovered this feature of Mockito: mock(Step.class, withSettings().extraInterfaces(NextStepStrategy.class));